manifests/vanilla/deploy-csi-snapshot-components.sh

```diff
@@ -31,13 +31,13 @@ Refer to https://kubernetes-csi.github.io/docs/snapshot-controller.html for furt
 
 Example command:
 
-./deploy-csi-snapshot-components.sh --release v4.1.1
+./deploy-csi-snapshot-components.sh --release v5.0.1
 
 usage: ${0} [OPTIONS]
 The following flags are required.
        --release        The external-snapshot release files to use.
-                        Default: v4.1.1
-                        Supported: v4.1.0, v4.1.1
+                        Default: v5.0.1
+                        Supported: v4.1.0, v4.1.1, v5.0.0, v5.0.1
```

Comment:
Not sure why we support 4.1.0 and 4.1.1 but not 4.2.0 and 4.2.1?

Reply:
afaik, it's the list of versions tested by our QE